site/content/exporting/zotero.lua

```diff
@@ -2046,4 +2151,13 @@ return {
   { Meta = Meta },
   { Cite = Cite_collect },
   { Cite = Cite_replace },
+  {
+    Para =
+      function (elem)
+        if elem.content[1].text == "{{bibliography}}" then
```

Comment:
Pandoc already has a facility for [placing the bibliography](https://pandoc.org/MANUAL.html#citations):

>  If the style calls for a list of works cited, it will be placed in a div with id refs, if one exists: ::: {#refs}
>  Otherwise, it will be placed at the end of the document.

Reply:
Resolving will take time.

Btw, the libreoffice plugin expects the 'banner' at the beginning of the document.  (I tried moving it to the end etc., but it wasnt' working).  _This_ is the reason why I you  see {{bibliography}} at the beginning of the markdown test case.

So, I need a recipe for how to inject the banner at the beginning of the document.  I have looked at `pandoc -C odt`, and I need to _dynamically_ inject this banner as the first of the body text in the AST.  I had difficulty figuring it out. Please help me with how to handle this.